introduction.md

```diff
@@ -1 +1,6 @@
-Hi, im edik! My favorite Code School path is JavaScript Best Practices.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?